spec/lib/hca/enrollment_system_spec.rb

```diff
@@ -909,6 +909,100 @@
     ]
   )
 
+  describe '#veteran_to_tera' do
+    test_method(
```

Comment:
I followed the pattern of using this test_method helper, but I have to be honest I think it makes reading and understanding these specs much more difficult (which is probably why it is hard to tell if scenarios are missing). It looks like this pattern predates everyone on the team as well. Looking at this code I think there could be some valuable refactoring for the specs and the `enrollment_system.rb` file to make it easier to understand and hopefully easier to update as needed.